Source/WebKit/Platform/IPC/Connection.h

```diff
@@ -777,12 +822,29 @@ void Connection::cancelReply(C&& completionHandler)
     }(std::make_index_sequence<std::tuple_size_v<typename T::ReplyArguments>> { });
 }
 
+template<typename T, typename C>
+Connection::AsyncReplyHandlerWithDispatcher Connection::makeAsyncReplyHandlerWithDispatcher(C&& completionHandler, RefCountedSerialFunctionDispatcher& dispatcher, ThreadLikeAssertion callThread)
+{
+    return AsyncReplyHandlerWithDispatcher {
+        {
+            [completionHandler = WTFMove(completionHandler), dispatcher = Ref { dispatcher }] (Decoder* decoder) mutable {
```

Comment:
this dispatcher ref is redundant and unused..

Reply:
see above.